packages/terra-core-docs/src/terra-dev-site/doc/status/About.1.doc.mdx

```diff
@@ -9,8 +10,16 @@ import StatusPropsTable from 'terra-status/lib/Status?dev-site-props-table';
 
 <Badge />
 
+[![Deprecated](https://badgen.net/badge/status/Deprecated/grey)](https://github.com/cerner/terra-core/tree/main/packages/terra-status#readme)
+
 # Terra Status
 
+<Notice variant="deprecation">
+
+`terra-status` is no longer supported. If you are using `terra-status`, consider replacing it with a supported component.
```

Comment:
What is the supported alternative? We should probably link it here.

Reply:
I agree with adding a supported component.

Reply:
Thanks for the feedback. This component does not meet accessibility criteria, and I am not aware of any alternative that we can point to. @eawww - do you have any suggestions for replacing terra-status? Is there an alternative or does the consumer need to work with UX on a different approach?

Reply:
@mjpalazzo @sdadn yeah this is an unfortunately common pattern in Cerner stuff that needs to go away. The alternative will largely depend on context but a nice plug-n-play replacement could often be iconography where meaning is conveyed by shape (and color if appropriate) and accessible alt-text.

I'd say it would usually be best to work with UX on a per-use-case basis.

Reply:
Thanks @eawww  So would terra-icon and terra-image be appropriate alternatives? These components have accessibility guides which provide guidance about alt-text, etc.

Reply:
Updated notice with reason, terra status does not comply with accessibility guidelines and alternatives.